astroquery/cadc/tests/DummyTapHandler.py

```diff
@@ -27,9 +27,9 @@ def check_method(self, method):
         if method == self.__invokedMethod:
             return
         else:
-            raise Exception("Method '"+str(method) +
-                            "' not invoked. (Invoked method is '" +
-                            str(self.__invokedMethod)+"')")
+            raise Exception("Method '"+str(method)
+                            + "' not invoked. (Invoked method is '"
```

Comment:
no need for the `+`, and it would be more cleaner to use an fstring instead.

Reply:
It turns out it was old dead code. Removed.